CRM/Contribute/BAO/ContributionRecur.php

```diff
@@ -902,6 +902,7 @@ protected static function isComplete($recurringContributionID, $installments) {
   public static function calculateRecurLineItems($recurId, $total_amount, $financial_type_id) {
     $originalContributionID = CRM_Core_DAO::getFieldValue('CRM_Contribute_DAO_Contribution', $recurId, 'id', 'contribution_recur_id');
     $lineItems = CRM_Price_BAO_LineItem::getLineItemsByContributionID($originalContributionID);
+    $lineSets = array();
     if (count($lineItems) == 1) {
```

Comment:
What if we change this line to ```if (count($lineItems) > 0) {``` instead of having separate elseif to handle the same!!